README.rst

```diff
@@ -1,6 +1,130 @@
-==========================
- REANA Demo - Hello World
-==========================
+====================================
+ REANA demo example - "hello world"
+====================================
 
-This repository provides a simple "hello world" example demonstrating data
-analysis encapsulation for running it via the REANA system.
+About
+=====
+
+This repository provides a simple "hello world" application example for `REANA
+<http://reanahub.io/>`_ reusable research data analysis plaftorm.
+
+Code
+====
+
+A simple "hello world" Python code for illustration:
+
+- `helloworld.py <helloworld.py>`_
+
+The program takes several optional arguments such as the name of the person to
```

Comment:
I think we should keep the description of the "program" as short as possible since the goal of this docs is giving an overview of the whole user experience with REANA